### ISSUE TITLE
cmd+q to quit 

### ISSUE THREAD

**Issue by bawigga** [limetext/lime#505]
On a mac, `cmd+q` is the norm for quitting apps. When I hit `cmd+q` I see log messages and I get a notice that we should be catching and handling the error appropriately.

```
[02/24/15 23:08:59] [DEBG] qmlfrontend.HandleInput: key=1000021, modifiers=4000000
2015/02/24 23:09:01 .:0: Signal QQmlEngine::quit() emitted, but no receivers connected to handle it.
```

**Comment by bawigga** [comment]
Some discussion about this on the go-qml mailing list: [Terminating things](https://groups.google.com/forum/#!searchin/go-qml/quit/go-qml/zH7gRswsfp4/tjhrwsnLGNgJ)

**Comment by bawigga** [comment]
For reference, I've [posted a question](https://groups.google.com/forum/#!topic/go-qml/ZKrM9Z-j_HQ) on the go-qml email list about this.

**Comment by zoli** [comment]
closed via #513
